### PR TITLE
WWW-67:Add staticquery and styles footer and copyright box

### PR DESCRIPTION
**Pull request by JuanJosephG**

### CODE REVIEW

File: src/components/organisms/Footer/Footer.js [hunk 1]
```diff
@@ -33,36 +32,61 @@ const Footer = (props) => {
...
+    imageUrl: footer.navBar.mainIcon.image.file.url,
+    slug: footer.navBar.mainIcon.to.slug,
+  }
 
   return (
-    <AppBar position="static" style={{ background: colorHeader }}>
+    <AppBar
+      position="relative"
+      justifyContent="center"
+      style={{
```
Comment (eacarras) [inline src/components/organisms/Footer/Footer.js]: I think we have to options here or we just use in all the app .scss files or we use the useStyle or makeStyle of Material UI. I think is better avoid send styles like here and also avoid the most that we can use `px`.

Comment (JuanJosephG) [reply src/components/organisms/Footer/Footer.js]: Yesterday I was trying to use **makestyles**. Something like:
```
  footer: {
    padding: theme.spacing(1),
    position: "fixed",
    bottom: 0,
  },
```
But It didn't work. Maybe we could do peer programming to analyze the options

Comment (eacarras) [reply src/components/organisms/Footer/Footer.js]: Sure!, just take into account for your next PR that is better to use the **makeStyles** option.

**Review by eacarras — CHANGES_REQUESTED** [review]
We will change the style that was added in the AppBar of this PR.

**Pull request merged**